design-principles.md

```diff
@@ -0,0 +1,83 @@
+
+You might be asking yourself - _wait, what are design principles?_ Don’t worry, they’re pretty simple!
+
+**Design principles** are a set of guidelines we refer to when designing (i.e. creating, maintaining or updating) something.
+
+In the context of [ethereum.org](http://ethereum.org) these design principles are the foundation for what we want the website to represent and project to the world. They’re both aspirational **and** functional. It’s not just how the website _looks_, but also how it _works_ and even how it makes someone _feel._ Everything, from the colors to the page layouts to how we talk about Ethereum on the website should be informed by these principles.
+
+**How does that work in practice? Let’s look at an example.** One of the principles is “Credible”, which means that we want visitors to the site to _feel_ and _know_ that the site is trustworthy - just like the wider Ethereum ecosystem. Within that principle, we have 3 functional “sub-principles” that we believe are actionable steps we can take to make the site credible:
+
+- _“Fresh”_ i.e. keep the content up-to-date.
+- _“Social Proof”_ i.e. show the size, diversity and activity of the ecosystem (you know: Eth2 progress, DeFi, gaming, all the hackathons, etc.)
+- _“Consistent”_ i.e. consistency in the design of the site and the tone and accuracy of the writing.
+
+So when we’re making design decisions, or copywriting decisions, we can then reference the “Credible” principle and ask:
+
+- _“Does the site reflect current information?”_
+- _“How and where are we showing the ecosystem’s size and activity?”_
+- _“Are the new proposed contributions by a community member that I’m reviewing consistent with the current design and writing on the site?”_
+
+## 🤔 How can you help?
+
+**Share your feedback on this document!** One of our proposed principles is “**Collaborative Improvement**” which means that we want the website to be the product of many contributors. So in the spirit of that principle, we want to share these design principles with the Ethereum community.
+
+While these principles are focused on the [ethereum.org](http://ethereum.org) website, we hope that many of them are representative of the values of the Ethereum ecosystem overall (e.g. you can see influence from the [principles of the Ethereum Whitepaper](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy)). Maybe you even want to incorporate some of them into your own project!
+
+---
+
+## Design Principles for Ethereum.org
+
+## 1. Inspirational
+
+The site should inspire users to dream of how Ethereum can change the world. It should motivate people to explore, play and tinker with the tools and apps of the Ethereum ecosystem.
+
+- **Radical:** The site should communicate Ethereum's ambitious goals to meaningfully change the world. It should be clear that Ethereum is not just some new product - it is a revolutionary technology.
```

Comment:
(Might seem like a nitpick, but I believe this sentence is important)

> not just some new product - it is a revolutionary technology

IMO, both "product" and "revolutionary technology" seem out of place here. 
"product" doesn't (even incorrectly describe) ethereum to me.
"revolutionary technology" seems to undermine all other disciplines (economy, sociology, networking, game theory, etc) needed for this stuff to work.

How about? 
"[...] that Ethereum is not just some new technology - it is a revolutionary set of ideas.
otherwise:
"[...] that Ethereum is not just some new technology - it's a deep technological shift.

Reply:
really helpful feedback @faraggi, thanks!

Can you tell me more about this point?

> "revolutionary technology" seems to undermine all other disciplines (economy, sociology, networking, game theory, etc) needed for this stuff to work.

that definitely isn't the intent of this wording - to me, "revolutionary technology" just means that the technology has large, significant implications - that it will change the world. It isn't intended to say that the underlying disciplines are unimportant.

What do you think about this option instead?

"[...] that Ethereum is not just some new tech stack - it is a transformational technology

Reply:
Re-reading my original critique, I've realized that it wasn't precise enough.

In any case, I personally like the new option much better. "Transformational" implies concrete changes to [society, tech, internet, etc].